src/config/config.js

```diff
@@ -0,0 +1,84 @@
+
+import dotenv from 'dotenv';
+
+dotenv.config(".env");
```

Comment:
```suggestion
import "dotenv/config"
```

Both lines can be done with ones